components/Spoiler.tsx

```diff
@@ -0,0 +1,24 @@
+import {Button} from "./Button";
```

Comment:
Poprosím nainstalovat rozšíření `Prettier` do VS Code (ještě musím dodělat automatické formátování na naší straně, aby to nebylo nutné).

Reply:
Prittier už aktivován